PostToExternalServices.py

```diff
@@ -0,0 +1,155 @@
+import json
+import logging
+import os
+
+import boto3
+import requests
+import tweepy
+
+AUTH0_BASE = "https://calligre.auth0.com"
+FB_BASE = "https://graph.facebook.com/v2.7"
+S3_BUCKET = "calligre-media"
+
+s3_client = boto3.client('s3')  # pylint: disable=C0103
```

Comment:
If this is staying in this repo, we should add `C0103` to the pylintrc (ie. have the same pylintrc as calligre/server).